src/modules/integrator-guide/pages/security-model/permissions-evaluation.adoc

```diff
@@ -0,0 +1,313 @@
+include::ROOT:partial$variables.adoc[]
+
+= Permission Evaluation
+
+The permission evaluation feature allows administrators to check which actions or operations can be performed by a user or a combination of roles.
+The evaluation entails comparing the different roles assigned to a user to determine the user's ability to perform the various operations within a resource.
+
+[NOTE]
+====
+xref:access-control/index.adoc[{PRODUCT_NAME} RBAC] uses the roles of the given user to determine if they have access to a given resource.
+====
+
+== How to evaluate applied permissions
+
+[NOTE]
+====
+Most of the {PRODUCT_NAME} resources allow you to define which xref:access-control/roles.adoc[roles] are allowed to access the resource.
+Locate the btn:[permissions] button in the user interface in order to manage the resource.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "security-model/access-control/general-permissions-button.png",
+    "alias": "general-permissions-button",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {
+    "x": 320,
+    "y": 0,
+    "h": 550,
+    "w": 1600
+  },
+  "focus": {
+    "x": 1466,
+    "y": 80,
+    "h": 31,
+    "w": 206
+  },
+  "annotations": [{
+    "kind": "box",
+    "x": 1536,
+    "y": 80,
+    "w": 134,
+    "h": 30
+  }]
+}
+----
+====
+
+Navigate to the desired resource, and click the btn:[permissions].
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "security-model/access-control/permission-button-page.png",
+    "alias": "new-user",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "kind": "box",
+    "padding": "xs",
+    "x": 1535,
+    "y": 75,
+    "w": 130,
+    "h": 42
+  }]
+}
+----
+
+Click the btn:[add +] button.
+A popup window with two dropdown options displays, the btn:[select role(s)] and btn:[search or select user].
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "security-model/access-control/add-role.png",
+    "alias": "add-role",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
```

Comment:
Annotation seems off aligned

![image](https://user-images.githubusercontent.com/12545711/198603386-ad21053f-2c38-4e55-b41a-465412716c99.png)

I'd place it either like this

![image](https://user-images.githubusercontent.com/12545711/198604249-a466504f-f08b-49d8-8e5f-d7bbad2db830.png)

or like this

![image](https://user-images.githubusercontent.com/12545711/198604564-948f976f-a7b8-4c27-92bc-0f100aabb9a4.png)